docs/Installation.md

```diff
@@ -45,8 +45,8 @@ The `gym-unity` subdirectory contains a package to interface with OpenAI Gym.
 
 ### Install Python and mlagents Package
 
-In order to use ML-Agents toolkit, you need Python 3.6.  
-[Download](https://www.python.org/downloads/) and install Python 3.6 if you do not already have it.
+In order to use ML-Agents toolkit, you need Python 3.6.1 or higher.  
+[Download](https://www.python.org/downloads/) and install Python 3.7 if you do not already have it.
```

Comment:
Should this say the "latest version of Python"?

Reply:
Done